components/Downloads/DownloadHeader/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import styles from './index.module.scss';
+import SectionTitle from '../../Common/SectionTitle';
+import type { NodeRelease } from '../../../types/releases';
+import type { FC } from 'react';
+
+type DownloadHeaderProps = {
+  release: NodeRelease;
+};
+
+const DownloadHeader: FC<DownloadHeaderProps> = ({ release }) => {
+  const intl = useIntl();
+  return (
+    <>
+      <div className={styles.downloadHeader}>
+        <SectionTitle
```

Comment:
IMHO SectionTitle should support the array items to be React Elements too, not just strings.

You could update the type of `SectionTitle` and then pass a `<FormattedMessage>` here. And remove the `useIntl`.